tests/manual-test-cases/Group13-vMotion/13-1-vMotion-VCH-Appliance.robot

```diff
@@ -0,0 +1,132 @@
+*** Settings ***
+Documentation  Test 13-1 - vMotion VCH Appliance
+Resource  ../../resources/Util.robot
+Suite Setup  Create a VSAN Cluster
+Suite Teardown  Run Keyword And Ignore Error  Kill Nimbus Server  %{NIMBUS_USER}  %{NIMBUS_PASSWORD}  *
```

Comment:
Not sure if the container log collection is part of the Teardown yet, its a nice to have during our nightly run log upload process.